GitExtensionsVSIX/source.extension.vsixmanifest

```diff
@@ -13,7 +13,9 @@
     </Metadata>
 
     <Installation>
-        <InstallationTarget Version="[14.0,15.0)" Id="Microsoft.VisualStudio.Community" />
+        <InstallationTarget Version="[14.0,16.0)" Id="Microsoft.VisualStudio.Community" />
```

Comment:
@azarkevich How should I go about rebasing on it? It doesn't appear to be part of the master.